VideoLocker/res/layout/popup_course_modes.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="168dp"
+    android:layout_height="wrap_content"
+    android:padding="16dp"
+    android:background="@drawable/white_rounded_bordered_bg">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textColor="@color/grey_3"
+        android:text="@string/course_view_modes" />
+
+    <TextView
+        android:id="@+id/videos_only"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="12dp"
+        android:padding="6dp"
+        android:drawablePadding="26dp"
+        android:clickable="true"
+        android:background="?android:attr/selectableItemBackground"
+        android:textColor="@color/course_modes_selected"
+        android:text="@string/course_view_mode_videos_only" />
+
+    <TextView
+        android:id="@+id/full_course"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="12dp"
+        android:padding="6dp"
+        android:drawablePadding="26dp"
+        android:clickable="true"
+        android:background="?android:attr/selectableItemBackground"
+        android:textColor="@color/course_modes_selected"
+        android:text="@string/course_view_mode_full_course" />
+</LinearLayout>
```

Comment:
Sure, I can add icons if provided. The text views just need to have `drawableLeft` and `drawableStart` attributes added.